library/core/class.form.php

```diff
@@ -134,6 +174,68 @@ public function addErrorClass(&$Attributes) {
         }
     }
 
+
+    /**
+     * Set the styles to use when outputting controls.
+     *
+     * @param string $name The name of the style. Currently this should be **legacy** or **bootstrap**.
+     * @return bool Returns **true** if the styles were set or **false** otherwise.
+     */
+    public function setStyles($name) {
+        if (isset($this->allStyles[$name])) {
+            $this->styles = $this->allStyles[$name];
+            return true;
+        } else {
+            $this->styles = $this->allStyles['legacy'];
+            return false;
+        }
+    }
+
+    /**
+     * Get a style element for the form.
+     *
+     * @param string $item The item, such as the element name or whatnot.
+     * @param null $default The default. If this isn't supplied then the "default" class will be returned.
+     * @return string Returns the element.
+     */
+    public function getStyle($item, $default = null) {
+        $item = strtolower($item);
+
+        if (isset($this->styles[$item])) {
+            return $this->styles[$item];
+        } elseif ($default !== null) {
+            return $default;
+        } elseif (isset($this->styles['default'])) {
+            return $this->styles['default'];
+        } else {
+            return '';
+        }
+    }
+
+    /**
+     * Translate old CSS classes using the style array.
+     *
+     * @param string|string[] $classes The classes to translate.
+     * @return string Returns the translated class string.
+     */
+    private function translateClasses($classes) {
```

Comment:
Okay I dealt with the issue and added your specific test case in a unit test. Thanks!